pkg/compose/build.go

```diff
@@ -48,7 +48,6 @@ func (s *composeService) Build(ctx context.Context, project *types.Project, opti
 }
 
 func (s *composeService) build(ctx context.Context, project *types.Project, options api.BuildOptions) error {
-	opts := map[string]build.Options{}
```

Comment:
Move it to closure below otherwise it could end up with `concurrent map write` panic. It occurs on my end when building multi service with buildx.

Reply:
Was spotted and merged in https://github.com/docker/compose/pull/10151, I have rebased accordingly.